runtime/vm/jvminit.c

```diff
@@ -6096,6 +6088,12 @@ protectedInitializeJavaVM(J9PortLibrary* portLibrary, void * userData)
 	}
 
 	/* Must be done after the compressed/full determination has been made */
+#ifdef J9VM_RAS_EYECATCHERS
+	J9RASInitialize(vm);
```

Comment:
Can this move this late?  Its being moved after the `if (JNI_OK != configureRasDump(vm)) {` call....

Reply:
This seems like it would make early crashes harder to debug.

Reply:
I think I understand the problem now - the major concern is whether other code will be trying to modify the `vm->j9ras` struct before this happens.

Reply:
Simply put, the VM does not crash that early in the real world, so I'm not concerned about that - all platforms sanity passes with these changes.

It might be better to move the parsing up (rather than the rest of it down), but I think that had problems the last time I tried. If you want, I'll try it again.

Reply:
The final decision will depend on heap sizes as well so I don't now that moving the parsing up will help unless that includes the GC's parsing as well.

I'm willing to accept this as is as I don't have a better suggestion